Bottom: a468773c9874dd8a769f4356b05f992e04289eb7
Top:    7f6907fb188abab268d7ca1a889162851cdce1c6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:02:38 +0000

Refresh of and-working-again-mostly

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 9b91a07..f4688ac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,7 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "hid/step/step_writer.h"
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
